src/compiler/commandLineParser.ts

```diff
@@ -379,6 +378,68 @@ namespace ts {
     }
 
     /**
+     * Parses non quoted strings separated by comma e.g. "a,b" would result in string array ["a", "b"]
+     * @param s
+     * @param existingValue
+     */
+    function parseMultiValueStringArray(s: string, existingValue: string[]) {
+        let value: string[] = existingValue || [];
+        let hasError = false;
+        let currentString = "";
+        if (s) {
+            for (let i = 0; i < s.length; i++) {

```

Comment:
This does't account for whitespace; so if I write

```shell
tsc --jsExtensions "js    ,     jsx    ,    es" foo.js bar.ts
```

then I'll get `["js    ", "     jsx    ", "    es"]`